html/semantics/embedded-content/the-iframe-element/iframe_sandbox_block_modals-5.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="uft-8">
+<title>iframe sandbox without allow_modals (document.execCommand("print"))</title>
+<link rel="author" title="Igalia" href="https://www.igalia.com/">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#attr-iframe-sandbox">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#the-iframe-element">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<iframe sandbox="allow-scripts"></iframe>
+<script>
+    let modaleName = "execCommandPrint";
```

Comment:
Where is it specified that execCommand("print") doesn't work?  I agree that it shouldn't, but if it's not specced yet, please file an issue or PR against an appropriate spec.

Reply:
Done in https://github.com/whatwg/html/issues/3549

Reply:
I also removed the test for that case.